[2023-02-10T14:17:24.810+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: cms_asp_noc.populate_all_tables manual__2023-02-10T13:53:02+00:00 [queued]>
[2023-02-10T14:17:24.820+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: cms_asp_noc.populate_all_tables manual__2023-02-10T13:53:02+00:00 [queued]>
[2023-02-10T14:17:24.820+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T14:17:24.821+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2023-02-10T14:17:24.822+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-10T14:17:24.831+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): populate_all_tables> on 2023-02-10 13:53:02+00:00
[2023-02-10T14:17:24.838+0000] {standard_task_runner.py:55} INFO - Started process 39047 to run task
[2023-02-10T14:17:24.842+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'cms_asp_noc', 'populate_all_tables', 'manual__2023-02-10T13:53:02+00:00', '--job-id', '1671', '--raw', '--subdir', 'DAGS_FOLDER/cms_upd.py', '--cfg-path', '/tmp/tmpbatysc5f']
[2023-02-10T14:17:24.844+0000] {standard_task_runner.py:83} INFO - Job 1671: Subtask populate_all_tables
[2023-02-10T14:17:24.905+0000] {task_command.py:388} INFO - Running <TaskInstance: cms_asp_noc.populate_all_tables manual__2023-02-10T13:53:02+00:00 [running]> on host 2886274f27ea
[2023-02-10T14:17:24.973+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cms_asp_noc
AIRFLOW_CTX_TASK_ID=populate_all_tables
AIRFLOW_CTX_EXECUTION_DATE=2023-02-10T13:53:02+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-10T13:53:02+00:00
[2023-02-10T14:17:24.975+0000] {sql.py:254} INFO - Executing: ['call cms.sp_upsert_asp_pricing();\ncall cms.sp_upsert_asp_ndc_hcpcs();\ncall cms.sp_upsert_noc_NDC_HCPCS();']
[2023-02-10T14:17:25.669+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-02-10T14:17:26.058+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-02-10T14:17:26.092+0000] {sql.py:375} INFO - Running statement: call cms.sp_upsert_asp_pricing();
call cms.sp_upsert_asp_ndc_hcpcs();
call cms.sp_upsert_noc_NDC_HCPCS();, parameters: None
[2023-02-10T14:17:26.097+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 266, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.InvalidSchemaName: schema "cms" does not exist
LINE 1: call cms.sp_upsert_asp_pricing();
             ^

[2023-02-10T14:17:26.119+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=cms_asp_noc, task_id=populate_all_tables, execution_date=20230210T135302, start_date=20230210T141724, end_date=20230210T141726
[2023-02-10T14:17:26.140+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1671 for task populate_all_tables (schema "cms" does not exist
LINE 1: call cms.sp_upsert_asp_pricing();
             ^
; 39047)
[2023-02-10T14:17:26.206+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-10T14:17:26.247+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
